project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc

```diff
@@ -600,11 +600,25 @@ function c4m_user_notifications_new_group_content_published($node) {
  * @param object $node
  *   Content node that was added.
  */
```

Comment:
Add the `$action` to the docs, explain the allowed values.

also fix the docs since it's now talking about a new content only